ts/features/bonus/bpd/store/reducers/details/activation/index.ts

```diff
@@ -69,6 +69,7 @@ export const bpdEnabledSelector = (
 
 /**
  * Return the Iban that the user has entered to receive the cashback amount
+ * @return {RemoteValue<string | undefined, Error>}
  */
 export const bpdIbanSelector = createSelector(
```

Comment:
Expressing the type of generics accepted by createSelector you can inform typescript about the value type returned
Although it is more verbose, as a developer I prefer to know immediately what is the [type returned by the selector](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md)

```typescript
export const bpdIbanSelector = createSelector<
  GlobalState,
  RemoteValue<string | undefined, Error>,
  RemoteValue<string | undefined, Error>
>
```

Reply:
Sure better!